srcpkgs/font-iosevka/template

```diff
@@ -12,8 +12,8 @@ distfiles="https://raw.githubusercontent.com/be5invis/Iosevka/v${version}/LICENS
  https://github.com/be5invis/Iosevka/releases/download/v${version}/ttc-iosevka-${version}.zip
  https://github.com/be5invis/Iosevka/releases/download/v${version}/ttc-iosevka-slab-${version}.zip"
 checksum="e61c0988bb231a321f14cce1b119a468f279ea86826c32e943ab16dbf08c1ba9
```

Comment:
I think this needs to be updated.

Reply:
You are right. I've updated the checksum. No idea why `xbps-src pkg` didn't complain on my machine though.

Reply:
I think it's because the file name isn't versioned, then it's fulfilled from your local cache. CI helps catch it :)

Reply:
That makes a lot of sense. Thanks for the explanation!